pkg/pool-manager/pool_test.go

```diff
@@ -414,4 +416,75 @@ var _ = Describe("Pool", func() {
 		})
 	})
 
-})
+	Describe("Multus Network Annotations API Tests", func() {
+		BeforeEach(func() {
```

Comment:
As noted offline, the before each should keep only the two following lines. poolManager needs to be initialized in the root of Describe and then populated in BeforeEach (we want to reset the manager per each test).

Reply:
Got it :)